gui/src/components/Notifications.js

```diff
@@ -25,14 +25,21 @@ export class Notifications extends React.Component {
   successPresenter = (success, i) => {
     // TODO: Update backend to have JSONAPI responses on Create to better discern between types.
     const isJob = success => success.initiators
+    const isJobRun = success => success.id
```

Comment:
To better future proof ourself, let's find a field that's more unique than `id`.

Reply:
I didnt like this either but post api returns only id

Reply:
@dimroc if you're going to retouch the api can we also return the job spec id that was run? Need it to generate <Link/> correctly in the notification

Reply:
I'll add it to the other PR.

Reply:
Here you go @NavyAdmiral: https://github.com/NavyAdmiral/chainlink/pull/1

The response creating a JobRun or BridgeType will now be the JSONAPI format of created object.